amplivelist/amp-live-list.go

```diff
@@ -50,7 +50,17 @@ type Page struct {
 
 var blogs []BlogItem
 
-func InitBlogs() {
+func Init() {
```

Comment:
Almost there. Still some duplication left:

* you're declaring the anonymous func three times
* duplicated string constants:
** dist folder
** sample category (which are changing soon and we have to replace all occurrences)
** sample names